include/send.h

```diff
@@ -93,6 +93,17 @@ extern void kill_client(struct Client *client_p, struct Client *diedie,
 extern void kill_client_serv_butone(struct Client *one, struct Client *source_p,
 				    const char *pattern, ...) AFP(3, 4);
 
+enum multiline_item_result {
+	MULTILINE_FAILURE = 0,
+	MULTILINE_SUCCESS = 1,
+	MULTILINE_WRAPPED = 2,
```

Comment:
No need to give explicit values here, IMO.

```suggestion
	MULTILINE_SUCCESS,
	MULTILINE_WRAPPED,
```